internal/clusterfeature/clusterfeaturedriver/transport_http.go

```diff
@@ -257,3 +261,16 @@ func isBadRequest(err error) bool {
 
 	return badRequest
 }
+
+func isNotFound(err error) bool {
+	notFound := false
+	errors.UnwrapEach(err, func(err error) bool {
+		if brErr, ok := err.(interface{ NotFound() bool }); ok {
```

Comment:
the `br` in `brErr` refers to it being a `BadRequest` error, so here it should be prefixed with `nf` or named using a better, error name independent scheme

Reply:
thx